etl-repair.md

```diff
@@ -123,4 +141,5 @@ In v1.104 of Kubecost, you may experience incorrect data display, such as costs
 5. After Asset and Allocation data has been repaired, wait an additional 30 minutes for federation to occur as a safeguard. Confirm the procedure has worked by validating some of the following:
    1. Query the last 7 days of data and observe reasonable unadjusted data.
    2. Check your storage bucket's `/federated/combined/etl/bingen/allocations/1d` and `/federated/combined/etl/bingen/assets/1d` directories to see that the files for impacted dates have been recently modified.
+   3. Review the federator's logs. Example shown above.
```

Comment:
Can we debrief when we capitalize the misc. federation terminology? I don't have the best grasp either but I'd like to be consistent across all our docs.

Reply:
@bstuder99 I've went ahead and capitalized all instances of "Federator" in this doc. I chose to capitalize because that's what Kubecost's logs show.